packages/main/src/Link.js

```diff
@@ -126,10 +126,46 @@ const metadata = {
 			defaultValue: "",
 		},
 
+		/**
+		 * Defines the aria-haspopup value of the component.
+		 *
+		 * @type String
+		 * @defaultvalue undefined
+		 * @private
+		 * @since 1.0.0-rc.16
```

Comment:
btw not sure if this is the proper version now?

Reply:
I'm not sure about this version. @ilhan007  could you check please?